software/excalidraw.yml

```
@@ -0,0 +1,11 @@
+name: excalidraw
+website_url: https://docs.excalidraw.com/
+description: Virtual hand-drawn style whiteboard. Collaborative and end-to-end encrypted. 
+licenses:
+  - MIT
+platforms:
+  - Nodejs
```

> https://hub.docker.com/r/excalidraw/excalidraw

With Docker being available, this should be added to the platforms.

```suggestion
platforms:
  - Nodejs
  - Docker
```

---

Thank you